### PR TITLE
Add CPU usage percentage to px

### LINKED ISSUE #45: CPU heading is misleading
Hi,

Thanks for this amazing piece of software, I've been playing arround with it 
and I found CPU heading misleading since it is display CPU time of the process, it would be nice
to have one more heading for CPU percentage usage with the name "CPU" and something
more accurate like "CPU TIME" for CPU time usage.

Regards

### PR DESCRIPTION
**Pull request by noahfx**
This will change:
-  CPU heading for CPU TIME
 
Will add:

-  CPU column with the CPU use percentage

This closes https://github.com/walles/px/issues/45

### CODE REVIEW

File: tests/px_terminal_test.py [hunk 1]
```diff
@@ -10,8 +10,8 @@
 def test_to_screen_lines_unbounded():
     procs = [testutils.create_process(commandline="/usr/bin/fluff 1234")]
     assert px_terminal.to_screen_lines(procs, None) == [
-        "  PID COMMAND USERNAME   CPU RAM COMMANDLINE",
-        "47536 fluff   root     0.03s  0% /usr/bin/fluff 1234"
+        "  PID COMMAND USERNAME CPU CPU TIME RAM COMMANDLINE",
+        "47536 fluff   root      0% 0.03s  0% /usr/bin/fluff 1234"
```
Comment (walles) [inline tests/px_terminal_test.py]: The spacing in this second line is mis-aligned with the heading. `/usr/...` should start right below `COMMANDLINE`. And having the CPU Time column being right-aligned would help readability IMO.

**Comment by walles** [comment]
@noahfx I like it! A colleague of mine said he was tricked by the multiple `%` columns, so I think this is a great change.

I wanted to modify the patch a bit but was unable to push to your branch, so I did them in #47 instead.

Can you review there please?

**Comment by walles** [comment]
Or you could do this:
```sh
git fetch --all
git cherry-pick 3757491d34ba564c66831e4459a360daf8990a3b
git cherry-pick 893b5715af47a309dc38687fac8a376a6dd5e235
```

Assuming you approve of my changes of course.

That's probably better, this way Github will credit you for the PR rather than me.

**Comment by noahfx** [comment]
Thanks! 
Those changes are OK with me, I have applied them to my PR branch 

P.S: Yesterday I uploaded the px deb package to the Debian Archive, it stills in the NEW Queue since its a new package it needs to be reviewed by the FTP Masters. https://ftp-master.debian.org/new/px_1.0.12-1.html

**Comment by walles** [comment]
Merged, thanks for your patch and for the Debian packaging status update :)

**Comment by walles** [comment]
@noahfx, I've been considering when to make another release with this change, but do you think I should wait until the Debian package has been approved by the FTP Masters?

**Comment by noahfx** [comment]
@walles Feel free to make the release, as soon as the package is approved I will upload the newer version

**Pull request merged**